src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -5,7 +5,11 @@ import uniqueId from 'lodash/uniqueId';
 import noop from 'lodash/noop';
 
 import { KEYS } from '../../constants';
+import ResponsiveMenuContainer from './ResponsiveMenuContainer';
+import { withMediaQuery } from '../media-query';
```

Comment:
This change introduces a new dependency on `react-responsive` to all existing imports of this component. This is a major breaking change and generally undesirable. We should consider moving the responsive-specific code into a new wrapping component so applications can opt-in to its use.